src/tools/spark-sstfile-generator/README.md

```diff
@@ -56,12 +61,13 @@ usage: nebula spark sst file generator
  -ti,--datasource_type_input <arg>           Currently supported source data types are hive|hbase|csv], and the default type is hive
 ```
 
-# Mapping file schema
+### Mapping file schema
 
 The format of the mapping files is json. File Schema is provided as [mapping-schema.json](mapping-schema.json) according to [Json Schema Standard](http://json-schema.org).
 Here is an example of mapping file: [mapping.json](mapping.json)
 
-# FAQ
+### FAQ
+
 ## How to use libnebula-native-client.so under CentOS6.5(2.6.32-431 x86-64)
```

Comment:
are you sure about the ## or ### ? It looks not logical

Reply:
There can only be one header with h1 #, changed `## How to use libnebula-native-client.so under CentOS6.5(2.6.32-431 x86-64)` to `#### How to use libnebula-native-client.so under CentOS6.5(2.6.32-431 x86-64)`. This is logical